### DISCUSSION TITLE
make toonsquid for pc aswell?

### DISCUSSION THREAD

**Discussion by yzhhhz** [keiwando/toonsquid#157] [Ideas]
i heard this app is pretty great but i dont have a ipad. so i was wondering if yall could import the app to pc or something

**Comment by St3fanos** [comment]
yes would be awoseme on surface devices too

**Comment by dabing1022** [comment]
Hope to support m1

**Comment by wpd2012** [comment]
Yeah, I need it running on my m1 mac first, then pc second. Desktop port will be huge, I don't mind paying again for the desktop version...

**Comment by fuki98** [comment]
agree with this
make sure its for windows..... i don't care about macs
